common/src/main/java/tachyon/worker/block/io/LocalFileBlockWriter.java

```diff
@@ -25,7 +25,6 @@
 import com.google.common.io.Closer;
```

Comment:
I also feel it is a bit weird to have them in worker but in common module. Do you have any good suggestion? I am open to the package.